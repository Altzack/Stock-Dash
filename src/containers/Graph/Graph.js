import React from 'react';
import styled from 'styled-components/macro';

const GraphContainer = styled.div`
  align-items: center;
  justify-content: center;
  padding: 200px;
  color: #fff;
  display: flex;
  background-color: #333;
  margin-bottom: 30px;
  margin-top: 30px;
  :hover {
    background-color: #333;
  }
  @media (min-width: 300px) {
    width: 300px;
    padding: 120px;
  }
  @media (min-width: 330px) {
    width: 340px;
  }
  @media (min-width: 600px) {
    width: 550px;
    padding: 170px;
  }
  @media (min-width: 900px) {
    width: 800px;
    padding: 200px;
  }
`;

const PageContainer = styled.div`
  display: flex;
  align-items: center;
  justify-content: space-evenly;
  width: 100%;
  flex-direction: column;
  flex-wrap: wrap;
`;

export default function Graph() {
  return (
    <PageContainer>
      <GraphContainer>graph</GraphContainer>
    </PageContainer>
  );
}
