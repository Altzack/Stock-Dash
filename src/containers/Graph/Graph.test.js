import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Graph from './Graph';

it('renders a graph by default', () => {
  const wrapper = shallow(<Graph />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
