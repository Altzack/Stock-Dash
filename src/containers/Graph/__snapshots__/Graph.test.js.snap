// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a graph by default 1`] = `
<Graph__PageContainer>
  <div
    style={
      Object {
        "marginTop": 150,
      }
    }
  >
    <Loader />
  </div>
</Graph__PageContainer>
`;
